_config.php

```diff
@@ -15,13 +15,13 @@
  */
 
 ShortcodeParser::get('default')
-	->register('file_link', array(File::class, 'handle_shortcode'))
-	->register('embed', array(EmbedShortcodeProvider::class, 'handle_shortcode'))
-	->register('image', array(Image::class, 'handle_shortcode'));
+	->register('file_link', [FileShortcodeProvider::class, 'handle_shortcode'])
```

Comment:
Leave EmbedShortcodeProvider in framework, since it's a non-assets module class.